edge-functions/next-news/components/header.tsx

```diff
@@ -0,0 +1,91 @@
+import Nav from "./nav";
+import Logo from "./logo";
+import Link from "next/link";
+
+export default () => (
```

Comment:
Let's export named components, Next.js should be complaining about this in the console.